### PR TITLE
Expose the global telemetry objects as sdk ones.

### PR DESCRIPTION
**Pull request by carlosalberto**
I remember @bogdandrutu asking for convenience getters this in the past.

### CODE REVIEW

File: sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java [hunk 1]
```diff
@@ -19,21 +19,18 @@
...
 import io.opentelemetry.OpenTelemetry;
-import io.opentelemetry.sdk.distributedcontext.DistributedContextManagerSdk;
-import io.opentelemetry.sdk.metrics.MeterSdk;
-import io.opentelemetry.sdk.trace.TracerSdk;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public class OpenTelemetryTest {
```
Comment (bogdandrutu) [inline sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java]: Is this file moved or copied?

Comment (carlosalberto) [reply sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java]: Renamed ;)

Comment (bogdandrutu) [reply sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java]: Super confusing, looks like a move in the review tool.

Comment (carlosalberto) [reply sdk/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java]: Yeah, agreed. I had the same problem when I re-reviewed my changes here ;)

**Pull request merged**